knowledge-base/v4-api.md

```diff
@@ -19,10 +19,11 @@ This document contains a collection of OData v4 features we want to implement an
 - Proposal accepted.
 
 ## $expand with Subqueries
-- Make queries on expanded entities (navigation properties)
+- Make queries on expanded entities (navigation properties and complex properties)
 - Works for `getAll` and `getByKey`
-- Example request for collection: https://services.odata.org/TripPinRESTierService/(S(ljgcbxqwp45c5l5m0h24kk1g))/People?$select=Friends&$expand=Friends($select=UserName,Emails;$filter=startswith(UserName,%27s%27))
-- Example request for single link: https://services.odata.org/TripPinRESTierService/(S(ljgcbxqwp45c5l5m0h24kk1g))/People?$select=BestFriend&$expand=BestFriend($select=UserName,Emails)
+- Example request for one to many link: https://services.odata.org/TripPinRESTierService/(S(ljgcbxqwp45c5l5m0h24kk1g))/People?$select=Friends&$expand=Friends($select=UserName,Emails;$filter=startswith(UserName,%27s%27))
+- Example request for one to one link: https://services.odata.org/TripPinRESTierService/(S(ljgcbxqwp45c5l5m0h24kk1g))/People?$select=BestFriend&$expand=BestFriend($select=UserName,Emails)
+- Could not find a working example for complex types
```

Comment:
Where did you pick up from that it's a possibility? the spec?

Reply:
I talked to the Java colleagues and now also crosschecked that in the spec: [Example 115](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_SystemQueryOptionexpand)